src/main/java/me/karakelley/http/handlers/RedirectHandler.java

```diff
@@ -0,0 +1,18 @@
+package me.karakelley.http.handlers;
+
+import me.karakelley.http.Request;
+import me.karakelley.http.Response;
+import me.karakelley.http.responses.MovedPermanently;
+
+public class RedirectHandler implements Handler {
+
+  public Response respond(Request request) {
+    Response response = new MovedPermanently();
+    return setResponse(request, response);
+  }
+
+  private Response setResponse(Request request, Response response) {
+    response.setHeaders("Location", "http://" + request.getHostAndPort() + "/");
```

Comment:
I _love_ how this is now only setting the header relevant to redirection and leaving the other, common headers to something else.